### ISSUE TITLE
Unknown job PurgeUnfinalizedAssetsJob

### ISSUE THREAD

**Issue by jerome2710** [solspace/craft3-freeform#293]
**Describe the bug or issue you're experiencing**
We are suddenly receiving dozens of error messages from various projects about a Freeform queue job being invalid. We are unsure what triggered it and if this is a bug in the plugin or something else. Please let us know if you need any additional information to investigate this issue.

![Schermafbeelding 2022-03-07 om 10 31 03](https://user-images.githubusercontent.com/4600095/157005113-059618ff-7dfc-4c7c-a5a4-b56873b03e25.png)

**Craft & Plugin Info:**

Various projects, all PHP 8, ranging from Craft CMS 3.7.30 to 3.7.35 and Freeform 3.13.0 to 3.13.2.

**Comment by kjmartens** [comment]
Sorry for the trouble you're experiencing, @jerome2710.

I am testing this out on my end and I can't seem to trigger this error. Perhaps your setup is a bit different from mine though. Would you be able to let us know how things are set up on your site, including which plugins you have installed, if you have any other routines happening that could be overlapping this kind of thing, etc? 🙂

**Comment by jerome2710** [comment]
Thank you for your reply, @kjmartens. While gathering information to allow debugging, we noticed the projects triggering these errors were using Redis. We use Redis for heavy sites, including the queue to reduce MySQL transactions. There seems to be a conflict in Redis databases, where jobs from project A (with Freeform) are handled by project B (without Freeform). We disabled Redis for these projects and have not received any errors since then.

Thank you for your reply anyways, this seems to be an issue from our side. We will investigate for a more robust solution.
